source/controlTypes.py

```diff
@@ -662,21 +664,50 @@ class OutputReason(Enum):
 REASON_CARET = OutputReason.CARET
 REASON_ONLYCACHE = OutputReason.ONLYCACHE
 
-#: Text to use for 'current' values. These describe if an item is the current item 
-#: within a particular kind of selection.
-isCurrentLabels: Dict[Union[bool, str], str] = {
+
+class IS_CURRENT(Enum):

```

Comment:
I must have missed this. I think it makes sense to follow Python conventions here, I believe naming should be:
```suggestion
class IsCurrent(Enum):
```

Upper case enum values are ok